opentelemetry-collector.yaml

```diff
@@ -1,7 +1,7 @@
 package:
```

Comment:
```suggestion
#nolint:valid-pipeline-git-checkout-commit
package:
```

Reply:
I think this will get the checkout pipeline to pass

Reply:
Is `nolint` preferable over `curl`ing the file? I made that change locally to test. This manifest file gets updated during release so it should match the latest version from the main repo.

Reply:
Oh I guess curling would make sense - in either case if something breaks we'll know about it, and curl saves on the whole repo clone